asm/field_control_avatar.s

```diff
@@ -1901,8 +1901,8 @@ _0806D780: .4byte 0x00004021
 _0806D784: .4byte gPlayerParty
 	thumb_func_end AdjustFriendship_step
 
-	thumb_func_start overworld_poison_timer_set
-overworld_poison_timer_set: @ 806D788
+	thumb_func_start DoPoisonFieldEffect_timer_set
```

Comment:
these functions should be all caps

Reply:
this one specifically looks like `ClearPoisonStepCounter` from pokeemerald and the next is `UpdatePoisonStepCounter`

Reply:
Yes, but I only did field_poison and ported the function names called by functions there. field_control_avatar.s is a large file and is related to `eventObjects` (which should be soon renamed and ported from pokeem by you) so I temporarily tend not to touch it.

Reply:
okay

Reply:
and sure I can correct DoPoisonFieldEffect_timer_set